src/state/slices/assetsSlice/assetStore.ts

```diff
@@ -0,0 +1,38 @@
+import { Asset, AssetDataSource, ChainTypes, NetworkTypes } from '@shapeshiftoss/types'
+import { createStore } from 'redux'
+
+export const initialAsset: Asset = {
+  caip2: '',
+  caip19: '',
+  chain: ChainTypes.Ethereum,
+  network: NetworkTypes.MAINNET,
+  symbol: '',
+  name: '',
+  precision: 18,
+  color: '',
+  secondaryColor: '',
+  icon: '',
+  sendSupport: true,
+  receiveSupport: true,
+  slip44: 60,
+  explorer: 'https://etherscan.io',
+  explorerTxLink: 'https://etherscan.io/tx/',
+  explorerAddressLink: '',
+  dataSource: AssetDataSource.CoinGecko,
+  description: ''
+}
+
+export function feedAssetStore(asset: Asset) {
+  return {
+    type: 'SET_ASSET',
+    asset
+  }
+}
+
+export function reducer(state = initialAsset, action: any) {
```

Comment:
take a look at the comment above about how to get the current asset id - we won't need a new global state slice just to manage this :)